test/acceptance/README.md

```diff
@@ -0,0 +1,243 @@
+# Acceptance Tests
+Future Farming and Countryside Programme - Demo Web Acceptance Tests 
+***
+
+This folder contains the acceptance tests for the FFC Demo Web service components as can be operated through the 'customer facing' `ffc-demo-web` front end microservice.
+
+The framework is (Cucumber)[https://cucumber.io/] and (webdriver.io)[https://webdriver.io/] based, containerised, expandable and based on the actively maintained webdriver.io Cucumber boilerplate project. 
+
+## Requirements
+
+- Docker Desktop 2.2.0.3 (42716) or higher
+- Node version 12 (not compatible with v13)
+  - if you have multiple versions of node, run `nvm use 12` to use version 12 for this alone
+
+# Quick start
+
+Docker is used to create a container for each of selenium-hub, chrome-browser and webdriver-cuke.
+* Selenium Hub allows concurrent execution of test cases 
+* Chrome Browser is the browser specified in the configuration file `wdio.conf.js` by default
+* Webdriver.io along with Cucumber is this framework that defines the tests.
+
+## How to run the tests
+
+1. Set the root URL for the environment in the environment variable `TEST_ENVIRONMENT_ROOT_URL`
+
+2. From the directory containing the dockerfile run `docker-compose up --build`. This will run an acceptance test against the FFC-Demo web service.
+
+3. The test report will be output to `./reports/html-reports`. Read more about report configuration in the [rpii/wdio-hmtl-reporter docs](https://github.com/rpii/wdio-html-reporter)
```

Comment:
Running the application does not create the folder for the reports, it's worth adding notes on creating the folder, and perhaps changing it's permission so Docker can write to it.

It may be simpler to have a single `html-reports` folder rather than create a hierarchy of `reports/html-reports`

Reply:
Added note to README as this only applies to WSL users, to create local dir